wander-embedded-sdk/src/components/iframe/wander-iframe.component.ts

```diff
@@ -73,18 +73,19 @@ export class WanderIframe {
     const { routeLayout } = options;
 
     if (typeof routeLayout === "string" || isRouteConfig(routeLayout)) {
-      // If a single value is passed, we use it for default and auth-requests. Anything else fallbacks to the default
-      // (currently modal):
+      // If a single value is passed, we use it for default, settings and auth-request. auth and account fallbacks to
+      // the default (currently modal):
 
       const defaultLayoutConfig = WanderIframe.getLayoutConfig(routeLayout);
 
       this.routeLayout = {
         default: defaultLayoutConfig,
+        settings: defaultLayoutConfig,
```

Comment:
By default, Settings now also uses the popup layout, or the same one defined for "default" if only one custom layout option is passed (`else` block below).